docs/smart-contracts/meebits_erc721.sol

```diff
@@ -0,0 +1,679 @@
+/**
+ *Submitted for verification at Etherscan.io on 2021-05-05
+*/
+
+pragma solidity 0.7.6;
+
+/**
+ *  __  __           _     _ _
+ * |  \/  |         | |   (_) |
+ * | \  / | ___  ___| |__  _| |_ ___
+ * | |\/| |/ _ \/ _ \ '_ \| | __/ __|
+ * | |  | |  __/  __/ |_) | | |_\__ \
+ * |_|  |_|\___|\___|_.__/|_|\__|___/
+ *
+ * An NFT project from Larva Labs.
+ *
+ */
+interface IERC165 {
+    function supportsInterface(bytes4 interfaceId) external view returns (bool);
+}
+
+interface IERC721 is IERC165 {
+    event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+    event Approval(address indexed owner, address indexed approved, uint256 indexed tokenId);
+    event ApprovalForAll(address indexed owner, address indexed operator, bool approved);
+    function balanceOf(address owner) external view returns (uint256 balance);
+    function ownerOf(uint256 tokenId) external view returns (address owner);
+    function safeTransferFrom(address from, address to, uint256 tokenId) external;
+    function transferFrom(address from, address to, uint256 tokenId) external;
+    function approve(address to, uint256 tokenId) external;
+    function getApproved(uint256 tokenId) external view returns (address operator);
+    function setApprovalForAll(address operator, bool _approved) external;
+    function isApprovedForAll(address owner, address operator) external view returns (bool);
+    function safeTransferFrom(address from, address to, uint256 tokenId, bytes calldata data) external;
+}
```

Comment:
interface IERC721 inherits the function supportsInterface. Solidty support multiple inheritance, but I guess the believed this simplified the contract